Frontend/android-native/mobility-common/src/main/java/in/juspay/mobility/common/MobilityCommonBridge.java

```diff
@@ -2527,7 +2685,10 @@ public void drawRouteV2 (final String drawRouteConfig) {
                     String endShortTitle = endMarkerConfig.optString("shortTitle", "");
                     String startTheme = startMarkerConfig.optString("theme", "LIGHT");
                     String endTheme = endMarkerConfig.optString("theme", "LIGHT");
-
+                    JSONObject sourceMarkerActionImageConfig = startMarkerConfig.optJSONObject("actionImage");
+                    JSONObject destMarkerActionImageConfig = endMarkerConfig.optJSONObject("actionImage");
```

Comment:
can we add this for stop markers as well.

Reply:
already addded